app/javascript/articles/components/TagList.jsx

```diff
@@ -1,16 +1,35 @@
 import { h } from 'preact';
 import { tagPropTypes } from '../../common-prop-types';
 
-export const TagList = ({ tags = [] }) => (
-  <div className="crayons-story__tags">
-    {tags.map((tag) => (
-      <a className="crayons-tag" href={`/t/${tag}`}>
-        <span className="crayons-tag__prefix">#</span>
-        {tag}
-      </a>
-    ))}
-  </div>
-);
+export const TagList = ({ tags = [], flare_tag }) => {
+  let tagsToDisplay = tags;
+  if (flare_tag) {
+    tagsToDisplay = tagsToDisplay.filter((tag) => tag !== flare_tag.name);
+  }
+  return (
+    <div className="crayons-story__tags">
+      {flare_tag && (
+        <a className="crayons-tag" href={`/t/${flare_tag.name}`}>
+          <span
+            className="crayons-story__flare-tag"
+            style={{
+              background: flare_tag.bg_color_hex,
+              color: flare_tag.text_color_hex,
+            }}
+          >
+            {`#${flare_tag.name}`}
```

Comment:
Can you wrap `#` with `<span className="crayons-tag__prefix">` similar to [how we do it with other tags](https://github.com/forem/forem/pull/10528/files#diff-bcb8e7ee1cc3e3133c3796a5a01689f0R26)?

Reply:
Sure thing!